libpod/define/diff.go

```diff
@@ -0,0 +1,26 @@
+package define
+
+// extra type to use as enum
+type DiffType uint8
+
+const (
+	// only diff containers
+	DiffContainer DiffType = 1 << iota
```

Comment:
If this is a bitfield, it seems like it would be better to manually specify - 0b1, 0b10, etc.

Reply:
`1 << iota` is basically just saying to shift the bit one step for each entry. It should be less error prone than writing this manually especially when you add new entries, However I understand that this syntax is confusing if you are not familiar with it. I can remove it if you want.